pkg/kancontroller/kancontroller.go

```diff
@@ -112,3 +120,17 @@ func isCACertMounted() bool {
 
 	return true
 }
+
+func createOrUpdateCRDs() bool {
+	createOrUpdateCRD := os.Getenv(createOrUpdateCRDEnvVar)
+	if createOrUpdateCRD == "" {
+		return false
+	}
+	c, err := strconv.ParseBool(createOrUpdateCRD)
+	if err != nil {
+		log.Print("environment variable", field.M{"CREATE_CRDS": createOrUpdateCRD})
+		return false
+	}
+
+	return c
```

Comment:
Could this be simplified to
```suggestion
	return createOrUpdateCRD == "true"
```